src/lib.rs

```diff
@@ -29,10 +29,33 @@ Since it's very simple we will start with limitations:
 - No unsafe.
 */
 
-#![doc(html_root_url = "https://docs.rs/simplecss/0.2.1")]
+// LINEBENDER LINT SET - lib.rs - v2
+// See https://linebender.org/wiki/canonical-lints/
+// These lints aren't included in Cargo.toml because they
+// shouldn't apply to examples and tests
+#![warn(unused_crate_dependencies)]
+#![warn(clippy::print_stdout, clippy::print_stderr)]
+// Targeting e.g. 32-bit means structs containing usize can give false positives for 64-bit.
+#![cfg_attr(target_pointer_width = "64", warn(clippy::trivially_copy_pass_by_ref))]
+// END LINEBENDER LINT SET
+#![cfg_attr(docsrs, feature(doc_auto_cfg))]
 #![cfg_attr(all(not(feature = "std"), not(test)), no_std)]
-#![forbid(unsafe_code)]
-#![warn(missing_docs)]
+#![doc(html_root_url = "https://docs.rs/simplecss/0.2.1")]
```

Comment:
I'm fairly sure that this is unnecessary, but maybe it does something important?

Reply:
Maybe [helps with linking types](https://users.rust-lang.org/t/you-do-not-need-html-root-url/73678)? We haven't needed it in any of our projects ever, so I tend to agree that it isn't needed. I'll remove it.